.github/workflows/nightly.yml

```diff
@@ -4,30 +4,8 @@ on:
     - cron: "10 * * * *"
```

Comment:
This is hopefully the final version of this file _EXCEPT_ for the crontab which should be scheduled once a day, not every hour (but I want it to run every hour during development of the test)